CMake/External_PROJ.cmake

```diff
@@ -1,11 +1,23 @@
 
+if(fletch_ENABLE_SQLite3)
+  get_system_library_name(sqlite3 sqlite_libname)
```

Comment:
BTW, what is the purpose of this?

Reply:
This call just abstracts away the if/else blocks for the different OS's and tacks the right things onto the name passed to give the real lib name. It doesn't work for everything since Johan made a bunch of assumptions about how things are named but it works for most libraries.

Reply:
Sure, but where/how is the result actually *used*?